# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/webidl/moz.build
# Commit: 1e56699898a2
# Full Hash: 1e56699898a27ae192ef2778c232ff88a4c53362
# Author: Paul Adenot <paul@paul.cx>
# Date: 2024-03-07 09:55:57
# Regressor Bug: 1858958
# File Overlap Count: 1
# Description:
#   Bug 1858958 - WebIDL file and skeleton code for AudioData. r=chunmin,webidl,saschanaz
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D190943
# ==============================================================================

diff -r 1590ebd92cf7 -r 1e56699898a2 dom/webidl/moz.build
--- a/dom/webidl/moz.build	Wed Mar 06 14:00:01 2024 +0000
+++ b/dom/webidl/moz.build	Wed Mar 06 14:00:12 2024 +0000
@@ -416,6 +416,7 @@
     "AudioBuffer.webidl",
     "AudioBufferSourceNode.webidl",
     "AudioContext.webidl",
+    "AudioData.webidl",
     "AudioDestinationNode.webidl",
     "AudioListener.webidl",
     "AudioNode.webidl",
